srcs/frontend/landing_page.html

```diff
@@ -0,0 +1,37 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+	<meta charset="UTF-8">
+	<meta name="viewport" content="width=device-width, initial-scale=1.0">
+	<title>Login Page</title>
+	<link rel="stylesheet" href="style.css">
+</head>
+<body>
+		<!-- <div id="elii"></div> -->
+		<!-- <div class="div1">
+			<svg xmlns="http://www.w3.org/2000/svg" id="icn1" fill="none">
+				<path d="M360.811 194.816C170.438 212.628 40.9481 72.3604 0 0L1213 4.87041V1004H1077.52C658.024 951.121 728.642 740.302 816.388 641.502C867.632 572.157 960.292 405.079 920.982 291.529C871.844 149.591 598.778 172.552 360.811 194.816Z" fill="#242B3E"/>
+			</svg>
+		</div> -->
+	<div class="btns">
+		<div id="login">
+			<div style="width: 219.53px; height: 60.85px; left: 85.29px; top: 2.65px; position: absolute; background: #F01E1E; box-shadow: 0px 4px 4px rgba(0, 0, 0, 0.25); border-radius: 24px"></div>
```

Comment:
We can make the css in separate file to make it more readable and easier to debug, however will accept this merge for this week